web/src/views/Apply/FormCircles.tsx

```diff
@@ -0,0 +1,11 @@
+import {CircleIcon} from "../../components/CircleIcon";
```

Comment:
If we re-export this component in `components/index.js`. We could do this
```suggestion
import {CircleIcon} from "~/components";
```

Reply:
cool